analysis_options.yaml

```diff
@@ -24,11 +24,6 @@ analyzer:
   strong-mode:
     implicit-dynamic: false
   errors:
-    # treat missing required parameters as a warning (not a hint)
-    missing_required_param: warning
```

Comment:
Did you intend to delete this (`missing_required_param`)? Previous to the last dart sdk roll, both `missing_required_param` and `missing_return` were treated as warnings.

Reply:
What's the difference?